cocotb/result.py

```diff
@@ -77,6 +77,8 @@ def __init__(self, retval):
 
 
 class TestComplete(Exception):
+    __test__ = False
```

Comment:
Not sure this is the right approach - I've found it's better to point pytest at your test directory rather than mark things all over your code as "not a test". Perhaps I'm missing something though.

Reply:
By default, it will look at this since name start with `Test` and it will cause a working. One way is to rename it other to leave the warning.